src/content/getting-started/create-workload-cluster/index.md

```diff
@@ -4,239 +4,315 @@ title: Creating a workload cluster and installing applications
+This will automatically use the latest release of the relevant Helm charts [cluster-aws](https://github.com/giantswarm/cluster-aws/blob/master/CHANGELOG.md) and [default-apps-aws](https://github.com/giantswarm/default-apps-aws/blob/master/CHANGELOG.md) (bundle of mandatory apps):
 
-```bash
-➜  kubectl get nodes
-NAME                                          STATUS   ROLES    AGE   VERSION
-ip-10-0-5-103.eu-central-1.compute.internal   Ready    worker   19h   v1.21.9
-ip-10-0-5-121.eu-central-1.compute.internal   Ready    master   19h   v1.21.9
-ip-10-0-5-14.eu-central-1.compute.internal    Ready    master   19h   v1.21.9
-ip-10-0-5-163.eu-central-1.compute.internal   Ready    master   19h   v1.21.9
-ip-10-0-5-183.eu-central-1.compute.internal   Ready    worker   19h   v1.21.9
-ip-10-0-5-5.eu-central-1.compute.internal     Ready    worker   19h   v1.21.9
-ip-10-0-5-53.eu-central-1.compute.internal    Ready    worker   19h   v1.21.9
+```sh
+# See hint about `--name` below!
+kubectl gs template cluster \
+  --provider capa \
+  --organization testing \
+  > cluster.yaml
 ```
 
-We can see that there are two kinds of roles: `master` and `worker`. Note that this still relates to the Management Cluster. In other words, the MC has some master nodes working on the cluster's "bureaucracy" and some worker nodes doing the actual work needed to manage the Workload Clusters.
+{{< /tab >}}
+{{< tab id="cluster-capz-azure-vms" title="CAPZ (Azure VMs)">}}
 
-The second command we suggest running is `kubectl get orgs`. This will show us the organizations that are defined in our installation.
+This will automatically use the latest release of the relevant Helm charts [cluster-azure](https://github.com/giantswarm/cluster-azure/blob/master/CHANGELOG.md) and [default-apps-azure](https://github.com/giantswarm/default-apps-azure/blob/master/CHANGELOG.md) (bundle of mandatory apps):
 
-```bash
-➜  kubectl get orgs
-NAME                    AGE
-production              97d
-testing                 97d
-giantswarm              97d
+```sh
+# See hint about `--name` below!
+kubectl gs template cluster \
+  --provider capz \
+  --organization testing \
+  --region westeurope \
+  --azure-subscription-id 00000000-0000-0000-0000-000000000000 `# fill in your subscription ID` \
+  > cluster.yaml
 ```
 
-You can see organizations as "buckets" for different entities. More information can be found [here]({{< relref "/platform-overview/organizations" >}}), but in general we can say that they are a way to isolate clusters, apps, etc. between different teams or environments.
+{{< /tab >}}
+{{< /tabs >}}
+
+If no name is specified for a workload cluster, a random one like `rfjh2` will be generated. We recommend you choose a naming scheme suiting your organization, and then stick to it (e.g. `dev-eu-1`, `prod-a`, `prod-ecommerce-b`). Add the `--name` parameter to specify the cluster name.
```

Comment:
are dashes allowed in cluster names? I bet they are not

Reply:
Right 😞 This isn't very flexible for customers. I understand the length limit because we fill the cluster name into other resource names, but why the character restriction? Is it because we do `strings.Split(name, "-")` in some operators?

Reply:
no idea

Reply:
Already discussed elsewhere and we'll solve this